### ISSUE TITLE
working fine if controlled by `MQTT` - web frontend looks blank

### ISSUE THREAD

**Issue by gretel** [hobbyquaker/airtunes2mqtt#1]
if topics `../enable` and `../volume` get published manually streaming via `TCP` socket works fine, awesome feature!

just the web-frontend looks empty in any case:
<img width="850" alt="screenshot 2018-09-28 at 10 06 05" src="https://user-images.githubusercontent.com/80815/46196018-328b2500-c306-11e8-89bc-3912f05ba359.png">

source is:
```
  | <!DOCTYPE html>
  |  
  | <title>airtunes2mqtt</title>
  |  
  | <meta name="mobile-web-app-capable" content="yes">
  |  
  | <meta name="viewport" content="width=device-width">
  | <meta name="apple-mobile-web-app-capable" content="yes" />
  | <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  |  
  | <link rel="stylesheet" href="style.css">
  | <script src="/socket.io/socket.io.js"></script>
  | <script src="../node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
  | <script type="module" src="airtunes-speaker.js"></script>
  |  
  | <div id="container">
  | <div id="left"></div>
  | <div id="content"></div>
  | <div id="right"></div>
  | </div>
  |  
  | <script src="script.js"></script>
  |  
```
the log does not show anything related.

any hints? regards

**Comment by hobbyquaker** [comment]
should be fixed with v1.2.3

**Comment by gretel** [comment]
@hobbyquaker looks like - thanks!
